main.go

```diff
@@ -1,107 +1 @@
-package main
 
-import (
-	"database/sql"
-	"fmt"
-	"log"
-	"os"
-	
-
-	_ "github.com/mattn/go-sqlite3"
-)
-
-func openDB(dbfile string) (*sql.DB) {
-	db, err := sql.Open("sqlite3", dbfile)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return(db)
-}
-
-func getHandleID(db *sql.DB, handle string) ([]int){
-	var handleID []int
-
-	rows, err := db.Query("SELECT rowid, id, service FROM handle where id=?", handle )
-	// fmt.Println("SELECT rowid, id, service FROM handle where id='%s'", handle)
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer 	rows.Close()
```

Comment:
this multiple spaces thing seems like something go fmt would have caught/fixed easily. I recommend running it. I code Go using VSCode + Go plugin and it comes with goreturns (which is go fmt plus organizing imports plus doing something return formatting) and it formats as you save. Try and use a plugin for your Go dev environment.